scripts/generate_export.py

```diff
@@ -1,15 +1,16 @@
 import os
 import sys
+from indico.client.client import IndicoClient
 from tqdm import tqdm
 
 import pandas as pd
 from collections import defaultdict
-from solutions_toolkit.auto_review import Reviewer, FieldConfiguration
+# from solutions_toolkit.auto_review import Reviewer, FieldConfiguration
```

Comment:
Why are we commenting this out? Reviewer is necessary for post processing.  Particularly in the YArdi use case. Check out line 355

Reply:
Yes, I understand. I commented it out because it was throwing an error on my end for some reason. It could not find Reviewer so for the time being for my test run, I commented it out. Please ignore this too! Gos! Sorry. I need to do a better job when pushing changes next time.